src/main/java/org/ow2/proactive/connector/iaas/model/Instance.java

```diff
@@ -66,4 +65,20 @@
 
     private String customScriptUrl;
 
+    @Override
+    public String toString() {
```

Comment:
I don't know why removing @ToString, but if you are trying to exclude credentials from it you can have something like this instead of redefining the toString method: 
@ToString(exclude = {"...", "..."})

Reply:
Would have much appreciated this, but I do not think this apply here: I attempts to mask sensitive data in the attribute of an attribute. I do not think this annotation can go that deep.

Reply:
which attribute are u hiding? I see u adding all attributes in this toString

Reply:
InfrastructureCrendentials > password. PR Updated in that way.